package.json

```diff
@@ -13,6 +13,7 @@
     "ng": "node --max_old_space_size=4096 ./node_modules/@angular/cli/bin/ng",
     "nx": "nx",
     "start": "nx serve",
+    "start:dev": "nx serve --host 0.0.0.0 --disableHostCheck",
```

Comment:
Mostly because its a potential security vulnerability for the running computer:
```
Warning: This is a simple server for use in testing or debugging Angular applications
locally. It hasn't been reviewed for security issues.

Binding this server to an open connection can result in compromising your application or
computer. Using a different host than the one passed to the "--host" flag might result in
websocket connection issues. You might need to use "--disableHostCheck" if that's the
case.
Warning: Running a server with --disable-host-check is a security risk. See https://medium.com/webpack/webpack-dev-server-middleware-security-issues-1489d950874a for more information.
```

Reply:
Yeah, not really. The server is not meant to run in production. That's what they're discouraging 

Basically, if you're running any firewall it is totally fine. Even then, it is likely fine.

Anything running from inside docker has to bind to any address as well because the containers are on different networks from the host and you wouldn't be able to use your dev service otherwise.